pulsetranslator/loghandler.py

```diff
@@ -128,6 +128,7 @@ def publish_build_message(self, data):
         for tag in data['tags']:
             if tag:
                 key_parts.append(tag)
+        key_parts.append(data['locale'])
```

Comment:
This has the potential to break users of pulsebuildmonitor.  I think we could just make this conditional to fix that problem, e.g.,

if data.get('locale'):  key_parts.append(data['local'])

Are there any disadvantages to that?